### ISSUE TITLE
Exception occurred, " openssh key type: ssh-rsa is not supported"

### ISSUE THREAD

**Issue by huobazi** [sshnet/SSH.NET#704]
When i create a scpclient 
```csharp
var client = new ScpClient("host", 22, "username", new PrivateKeyFile("~/.ssh/id_rsa", "123123"));
```

I got a exception:
> openssh key type: ssh-rsa is not supported

The stack trace:
```csharp
System.AggregateException: One or more errors occurred. (openssh key type: ssh-rsa is not supported)
 ---> Renci.SshNet.Common.SshException: openssh key type: ssh-rsa is not supported
   at Renci.SshNet.PrivateKeyFile.ParseOpenSshV1Key(Byte[] keyFileData, String passPhrase)
   at Renci.SshNet.PrivateKeyFile.Open(Stream privateKey, String passPhrase)
   at Renci.SshNet.PrivateKeyFile..ctor(String fileName, String passPhrase)
```

I found the exception source may be came from:

https://github.com/sshnet/SSH.NET/blob/546e2b9ece47f1982811a0bcdafa93fec7c5d0e3/src/Renci.SshNet/PrivateKeyFile.cs#L415

```bash
┌[mw82@Marble-MacBook-Pro.local]-(~)
└> ssh -Q key
ssh-ed25519
ssh-ed25519-cert-v01@openssh.com
ssh-rsa
ssh-dss
ecdsa-sha2-nistp256
ecdsa-sha2-nistp384
ecdsa-sha2-nistp521
ssh-rsa-cert-v01@openssh.com
ssh-dss-cert-v01@openssh.com
ecdsa-sha2-nistp256-cert-v01@openssh.com
ecdsa-sha2-nistp384-cert-v01@openssh.com
ecdsa-sha2-nistp521-cert-v01@openssh.com
┌[mw82@Marble-MacBook-Pro.local]-(~)
```

**Any response was appreciated**

**Comment by darinkes** [comment]
There is an PR for those Keys: https://github.com/sshnet/SSH.NET/pull/614

**Comment by theKidOfArcrania** [comment]
Hi, when will there be a release version supporting this PR?

**Comment by Rob-Hague** [comment]
Fixed by #614 which is in version 2023.0.0
